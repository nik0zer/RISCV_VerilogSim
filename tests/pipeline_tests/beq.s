.section .text
.global _start

_start:
    addi x1, x0, 1
    addi x2, x0, 5
    addi x3, x0, 5
loop:
    nop
    addi x1, x1, 1
    beq x2, x3, loop
    nop
    nop
    nop
    nop
    nop
    nop
    nop
    nop
    nop
    nop
    nop
    nop
    nop
    nop
    nop