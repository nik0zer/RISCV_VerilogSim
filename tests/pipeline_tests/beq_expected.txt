x
x
x
x
0000000000000001
0000000000000005
0000000000000005
0000000000000002
x
x
x
0000000000000003
x
x
x
0000000000000004
x
x
x
0000000000000005
x
x
x
0000000000000006
x
x
x
